go.mod

```diff
@@ -14,28 +14,28 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/shirou/gopsutil v3.21.11+incompatible
 	github.com/sirupsen/logrus v1.9.3
-	github.com/spf13/afero v1.9.5
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.16.0
 	github.com/stretchr/testify v1.8.4
 	github.com/tklauser/go-sysconf v0.3.10 // indirect
 	github.com/trento-project/contracts/go v0.0.0-20230823130307-95ed2147fa9d
-	github.com/vektra/mockery/v2 v2.32.3
+	github.com/vektra/mockery/v2 v2.36.0
 	github.com/wagslane/go-rabbitmq v0.10.0
-	golang.org/x/sync v0.3.0
+	golang.org/x/sync v0.4.0
 	google.golang.org/protobuf v1.31.0
 )
 
 require (
 	github.com/d-tux/go-fstab v0.0.0-20141204152952-eb4090f26517
 	github.com/iancoleman/strcase v0.2.0
+	github.com/spf13/afero v1.9.5
 	github.com/tredoe/osutil v1.0.6
 	golang.org/x/mod v0.9.0
 )
 
 require (
-	github.com/chigopher/pathlib v1.0.0 // indirect
+	github.com/chigopher/pathlib v0.15.0 // indirect
```

Comment:
Going back in version?
Strange. Please have a look on this and try to get a clean update of the file

Reply:
Missed this non-requested change, I guess from `go mod`, removed (as well as the mockery one)